packages/web-components/src/components/masthead/masthead-menu-button.ts

```diff
@@ -54,6 +102,23 @@ class DDSMastheadMenuButton extends BXHeaderMenuButton {
   @property({ reflect: true })
   slot = 'brand';
 
+  connectedCallback() {
+    super.connectedCallback();
+
+    const mqMedium = this.ownerDocument?.defaultView?.matchMedia(`(max-width: ${breakpoints.md.width})`);
+    this._responsiveMd = mqMedium ? mqMedium.matches : false;
+
+    this._mqHandler = on(mqMedium, 'change', this._handleMediaQuery.bind(this));
+  }
+
+  disconnectedCallback() {
+    if (this._mqHandler) {
+      this._mqHandler.release();
```

Comment:
```suggestion
      this._mqHandler = this._mqHandler.release();
```

This codebase nulls out handle properties when they are no longer in use.